<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHMPendu</class>
 <widget class="QMainWindow" name="IHMPendu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Jeu du pendu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>611</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="JouerBut">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>JOUER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConfigBut">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Config.</string>
    </property>
   </widget>
   <widget class="QLabel" name="JeuduPenduLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Jeu du pendu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="FichierMots">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>451</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fichier de mots :</string>
    </property>
   </widget>
   <widget class="QLabel" name="MotsDispo">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>510</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mots disponible:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameLine">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nomduJoueurLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom du joueur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="PhotoErreur">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>201</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QLabel" name="MotaTrouverLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot Ã  trouver :</string>
    </property>
   </widget>
   <widget class="QLabel" name="MotCacher">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nombreErreur">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Erreur</string>
    </property>
   </widget>
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
